test/OpenProject.Tests/WebViewIntegration/OpenProjectInstanceValidatorTests.cs

```diff
@@ -15,8 +15,8 @@ public class IsValidOpenProjectInstanceAsync
       [InlineData("https://wieland.openproject.com/api/v3", "https://wieland.openproject.com")]
       [InlineData("https://community.openproject.org/api/v3/", "https://community.openproject.org")]
       [InlineData("https://wieland.openproject.com/api/v3/", "https://wieland.openproject.com")]
-      [InlineData("https://community.openproject.org:443/api/v3", "https://community.openproject.org:443")]
-      [InlineData("https://wieland.openproject.com:443/api/v3", "https://wieland.openproject.com:443")]
```

Comment:
as 443 is the default port for https, and we use the `Uri` type, it doesn't write default ports to string. Neither would I expect them to appear in the list.